crates/bevy_math/src/rects/irect.rs

```diff
@@ -180,11 +183,14 @@ impl IRect {
 
     /// The center point of the rectangle.
     ///
+    /// # Rounding Behaviour
+    /// If the (min + max) contains odd numbers they will be **rounded down** to the nearest whole number when calculating the center.
```

Comment:
We're already dedicating a whole top-level section to rounding, I don't think we need to further highlight. Let's keep things simple.

```suggestion
    /// If the (min + max) contains odd numbers they will be rounded down to the nearest whole number when calculating the center.
```